Pipfile

```diff
@@ -13,6 +13,7 @@ aiohttp = "==3.8.3"
 requests = "==2.28.1"
 tomli = "==2.0.1"
 jinja-cli = "==1.2.2"
+pandas = "==1.5.1"
```

Comment:
Although it looks like you're using a good chunk of the pandas functionality here, so maybe not

Reply:
Actually, one concern is that pandas will load the entire file to memory, which might be bad

Reply:
I will stick with pandas for now. It's so just so easy to use.

Reply:
Fair enough - but you're sure memory usage won't be an issue? Those metadata files 315K x 112 entries so not small

Reply:
I think it will be okay. I have used pandas to load larger files. We probably won't need all 112 features. So far I am aware of ~10 stats that we need. We probably will add more. But I guess at this stage I will stick with pandas until we figure out the whole scope.

Reply:
:+1: